examples/tutorials/audio_resampling_tutorial.py

```diff
@@ -230,26 +146,32 @@ def benchmark_resample(
 # plotted waveform, and color intensity the amplitude.
 #
 
-
 sample_rate = 48000
-resample_rate = 32000
-
 waveform = get_sine_sweep(sample_rate)
+
 plot_sweep(waveform, sample_rate, title="Original Waveform")
-play_audio(waveform, sample_rate)
+Audio(waveform.numpy()[0], rate=sample_rate)
 
+######################################################################
+#
+# Now we resample (downsample) it.
+#
+# We see that in the spectrogram of the resampled waveform, there is an
+# artifact, which was not present in the original waveform.
+
+resample_rate = 32000
 resampler = T.Resample(sample_rate, resample_rate, dtype=waveform.dtype)
 resampled_waveform = resampler(waveform)
-plot_sweep(resampled_waveform, resample_rate, title="Resampled Waveform")
-play_audio(waveform, sample_rate)
 
+plot_sweep(resampled_waveform, resample_rate, title="Resampled Waveform")
+Audio(resampled_waveform.numpy()[0], rate=resample_rate)
 
 ######################################################################
-# Controling resampling quality with parameters
-# ---------------------------------------------
+# 3. Controling resampling quality with parameters
+# ------------------------------------------------
 #
-# Lowpass filter width
-# ~~~~~~~~~~~~~~~~~~~~
+# 3.1. Lowpass filter width
```

Comment:
Similar to my comment in data augmentations PR: I prefer subsections in indented form over subsection numbers. I think this organization is a bit harder to read and also places the same amount of emphasis on the subsections as the main sections

<img width="199" alt="Screen Shot 2022-05-16 at 10 15 39 AM" src="https://user-images.githubusercontent.com/16568633/168613308-70716547-ef22-46e9-84d6-c7c0c9c8a9b1.png">